### PR TITLE
Slackbot Authentication

### PR DESCRIPTION
**Pull request by NicolasAndruzzi**
This Pull Request is for Implementing Authorization with our LocoBot

At the beginning of the application, local variables are set to equal each of the three heroku defined environment variables.  There is a token for each of the three routes

-SLACK_MAIN_TOKEN is for the main route (used for catching all posts of user entry data)
-SLACK_WHERE_TOKEN is for the /LocoWhere route
-SLACK_WHO_TOKEN is for the /LocoWho route

each variable is then checked to see that none of them are undefined
(IF THEY'RE UNDEFINED: THROW ERROR)(EXPLODE APP)

Each of the three post routes then checks internally whether their respective token matches the token of the requester (which we expect to match)

The bottom line = The app is now more robust and safe
-------------------------------------------------------------------------
arnold

### CODE REVIEW

**Comment by pipermerriam** [comment]
@NicolasAndruzzi this needs to be rebased

**Comment by pipermerriam** [comment]
👍
